packages/terra-navigation-layout/tests/jest/__snapshots__/NavigationLayout.test.jsx.snap

```diff
@@ -64,12 +64,14 @@ exports[`NavigationLayout should render a NavigationLayout with provided compone
 <Layout
   header={
     <Header
+      app={undefined}
```

Comment:
Should app be defined, is this just a new output from updating jest?

Reply:
New output from jest v3